.github/workflows/gradle.yml

```diff
@@ -0,0 +1,16 @@
+name: build-ci
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Java setup
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Build
+      run: ./gradlew setupDecompWorkspace && ./gradlew build
```

Comment:
It doesn't seem to upload artifacts either

Reply:
I can fix that as well, one moment